config/routes.rb

```diff
@@ -191,6 +194,9 @@
   end
   resources :case_court_orders, only: %i[destroy]
 
+  resources :additional_expenses, only: %i[create destroy],
+    constraints: lambda { |req| req.format == :json }
+
```

Comment:
I did these as root-level resources. They could be case_contact member resources (nested), as they can't exist without a contact parent. I tend to error on the side of not nesting resources... and it was easier. Can move if need be.

The constraints for :json was new to me. Made sense for the no-view / api style endpoint.